server/database/dummy-data-prod/stepsContent.js

```diff
@@ -0,0 +1,538 @@
+        ],
+      },
+      {
+        title: `Housing costs`,
+        description: `You can get this from your landlord or mortgage provider.`,
+        thisCanInclude: [
+          `How much rent you are charged and how often`,
+          `How much service charges you are charged and how often`,
+          `Mortgage payments`,
+          `Ground rent`,
+        ],
+        tips: [],
+      },
+      {
+        title: `Childcare costs`,
+        description: ``,
+        thisCanInclude: [],
+        tips: [],
+      },
+      {
+        title: `Income details of any other adults living in the property that are not lodgers e.g. grown up children, elderly parents`,
+        description: ``,
+        thisCanInclude: [],
+        tips: [],
+      },
+    ],
+    topTip: `The more accurate the information you give the more accurate the estimate will be.`,
+    otherTips: [
+      `Keep hold of any documents you dig out to help you work out the bits you need to know for the calculator. These are often needed in the application so having them stored can be helpful to make things quicker!`,
+    ],
+  };
+
+  // CLAIMING
+  const createAccountContent = {
+    stage: 'CLAIMING',
+    stepOrder: 2,
+    title: `Create account`,
+    description: `Create an account on the Government website if you haven’t already. You’ll want to do this as soon as possible!`,
+    pageTitle: ``,
+    pageDescription: ``,
+    howLongDoesItTake: { timeRangeText: `5 to 10 minutes` },
+    WhereDoYouNeedToGo: {
+      link: 'https://www.universal-credit.service.gov.uk',
+      type: 'LINK',
+      title: `Create account`,
+    },
+    thingsYouWillNeed: [
+      {
+        title: `Email address`,
+        description: `This should be the bank account that your Universal Credit will be paid into. You can set up a free bank account in most high street banks or an online bank if you don’t already have one.`,
+        thisCanInclude: [],
+        tips: [],
+      },
+      { title: `Bank account`, description: ``, thisCanInclude: [], tips: [] },
+      {
+        title: `Access to your mobile phone (if you have one)`,
+        description: ``,
+        thisCanInclude: [],
+        tips: [],
+      },
+      {
+        title: `Your partner's Universal Credit linking code (if you have a partner and they have one)`,
+        description: ``,
+        thisCanInclude: [],
+        tips: [
+          `If with a partner BOTH will need to do this separately and then the accounts will need to be ‘LINKED’ via a linking code that will be given on the screen during this process.`,
+        ],
+      },
+    ],
+    whatYouWillNeedToKnow: [
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+      { title: ``, description: ``, thisCanInclude: [], tips: [] },
+    ],
```

Comment:
Just check that this is meant to be empty like that as there is no mentioning it in the doc file

Reply:
good spot - fixed